tests/org.eclipse.n4js.ide.tests/src/org/eclipse/n4js/ide/tests/xtext/server/ChunkedResourceDescriptionsTest.java

```diff
@@ -0,0 +1,219 @@
+/**
+ * Copyright (c) 2020 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+package org.eclipse.n4js.ide.tests.xtext.server;
+
+import java.util.Collections;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.emf.ecore.EcorePackage;
+import org.eclipse.n4js.ide.xtext.server.util.IndexedChunkedResourceDescriptions;
+import org.eclipse.xtext.naming.QualifiedName;
+import org.eclipse.xtext.resource.IEObjectDescription;
+import org.eclipse.xtext.resource.impl.ChunkedResourceDescriptions;
+import org.eclipse.xtext.resource.impl.ResourceDescriptionsData;
+import org.eclipse.xtext.resource.persistence.SerializableEObjectDescription;
+import org.eclipse.xtext.resource.persistence.SerializableResourceDescription;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import com.google.common.base.Stopwatch;
+import com.google.common.collect.Iterables;
+import com.google.common.math.StatsAccumulator;
+
+/**
+ * Performance test for resource description implementations.
+ */
+@Ignore("TODO Implement a way to make this a real test with assertions on the timing")
```

Comment:
Regarding `@Ignore("TODO` - Should we ignore or do it?

Reply:
I'll implement it but wanted to ignore the test for now because it does some sysouts.